src/EditorFeatures/CSharpTest/CodeActions/ReplaceMethodWithProperty/ReplaceMethodWithPropertyTests.cs

```diff
@@ -1898,25 +1898,16 @@ int IGoo.Goo
 }");
         }
 
-        [WorkItem(443523, "https://devdiv.visualstudio.com/DevDiv/_workitems?id=443523")]
+        [WorkItem(23593, "https://github.com/dotnet/roslyn/issues/23593")]
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsReplaceMethodWithProperty)]
         public async Task TestMetadataOverride()
         {
-            await TestWithAllCodeStyleOff(
+            await TestMissingAsync(
 @"class C : System.Type
 {
     public override int [||]GetArrayRank()
     {
     }
-}",
-@"class C : System.Type
-{
-    public override int {|Warning:ArrayRank|}
```

Comment:
❓ What does Warning mean here?

❓ Is it possible to implement this refactoring such that a conflict annotation is placed on the definition of `ArrayRank` (the property)?

Reply:
```{|Warning:|}``` and ```{|Conflict:|}``` refer to annotations that providers can place on pieces of syntax.  These annotations will then show up in the preview UI in two forms.  First, the syntax in question will be highlighted in yellow/red respectively.  Also, the warning/conflict message will show up below the preview.  

My ideal would be that the glyph of a code-action could also present if there were warnings/conflicts to be aware of as well. However, that might be complex to provide if the detection of hte problem only happens during application of an expensive fix.